### PR TITLE
LPS-91820 The field options of conditions must not contain the Paragraph field, only showing the paragraph field on actions target list

### PR DESCRIPTION
**Pull request by alinedoleron**

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by brunobasto** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleEditor/RuleEditor.es.js [hunk 1]
```diff
@@ -164,25 +164,9 @@ class RuleEditor extends Component {
...
-						)
-					),
-					type: Config.string(),
-					value: Config.string()
-				}
-			)
-		).internal().valueFn('_fieldOptionsValueFn'),
+		fieldOptions: Config.arrayOf(fieldOptionStructure).internal().valueFn('_fieldOptionsValueFn'),
+
+		fieldOptionsWithRestrictions: Config.arrayOf(fieldOptionStructure).internal().valueFn('_fieldOptionsWithRestrictionsValueFn'),
```
Comment (brunobasto) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleEditor/RuleEditor.es.js]: @alinedoleron I don't understand why we have both fieldOptions and fieldOptionsWithRestrictions. The restriction could be just the way fieldOptions work.

**Comment by alinedoleron** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**